File: redome-front/src/app/paciente/avaliacao/resposta.pendencia.dto.ts
/**
 * Classe que representa uma resposta RespostaPendenciaDTO dto
 * 
 * @author Cintia Oliveira
 * @export
 */
export class RespostaPendenciaDTO {

	private _resposta: string;
	private _usuario:string;
	private _exame: number;
	private _dataFormatadaDialogo:string;
	private _evolucao: number;
	private _link:string;
	private _estiloIcone:string;

	/**
	 * 
	 * 
	 * @type {string}
	 * @memberof RespostaPendenciaDTO
	 */
	public get resposta(): string {
		return this._resposta;
	}

	/**
	 * 
	 * 
	 * @memberof RespostaPendenciaDTO
	 */
	public set resposta(value: string) {
		this._resposta = value;
	}


	/**
	 * 
	 * 
	 * @type {Usuario}
	 * @memberof RespostaPendenciaDTO
	 */
	public get usuario(): string {
		return this._usuario;
	}

	/**
	 * 
	 * 
	 * @memberof RespostaPendenciaDTO
	 */
	public set usuario(value: string) {
		this._usuario = value;
	}
	
	/**
	 * 
	 * 
	 * @type {string}
	 * @memberof RespostaPendenciaDTO
	 */
	public get dataFormatadaDialogo(): string {
		return this._dataFormatadaDialogo;
	}

	/**
	 * 
	 * 
	 * @memberof RespostaPendenciaDTO
	 */
	public set dataFormatadaDialogo(value: string) {
		this._dataFormatadaDialogo = value;
	}
	
	public get exame(): number {
		return this._exame;
	}

	public set exame(value: number) {
		this._exame = value;
	}

	public get evolucao(): number {
		return this._evolucao;
	}

	public set evolucao(value: number) {
		this._evolucao = value;
	}

	public get link(): string {
		return this._link;
	}

	public set link(value: string) {
		this._link = value;
	}

	public get estiloIcone(): string {
		return this._estiloIcone;
	}

	public set estiloIcone(value: string) {
		this._estiloIcone = value;
	}
	
}

File: redome/src/main/java/br/org/cancer/modred/controller/dto/RespostaPendenciaDTO.java
package br.org.cancer.modred.controller.dto;

import java.time.LocalDateTime;

import com.fasterxml.jackson.annotation.JsonIgnore;
import com.fasterxml.jackson.annotation.JsonRootName;

import br.org.cancer.modred.util.DateUtils;

/**
 * DTO para exibição da dialogo da pendencia.
 * 
 * @author Cintia Oliveira
 *
 */
@JsonRootName("respostaPendencia")
public class RespostaPendenciaDTO {

	private String resposta;
	private String usuario;
	@JsonIgnore
	private LocalDateTime data;
	private Long exame;
	private Long evolucao;

	/**
	 * @return the resposta
	 */
	public String getResposta() {
		return resposta;
	}

	/**
	 * @param resposta the resposta to set
	 */
	public void setResposta(String resposta) {
		this.resposta = resposta;
	}

	/**
	 * @return the usuario
	 */
	public String getUsuario() {
		return usuario;
	}

	/**
	 * @param usuario the usuario to set
	 */
	public void setUsuario(String usuario) {
		this.usuario = usuario;
	}

	/**
	 * @return the data
	 */
	public LocalDateTime getData() {
		return data;
	}

	/**
	 * @param data the data to set
	 */
	public void setData(LocalDateTime data) {
		this.data = data;
	}

	/**
	 * @return the exame
	 */
	public Long getExame() {
		return exame;
	}

	/**
	 * @param exame the exame to set
	 */
	public void setExame(Long exame) {
		this.exame = exame;
	}

	/**
	 * @return the evolucao
	 */
	public Long getEvolucao() {
		return evolucao;
	}

	/**
	 * @param evolucao the evolucao to set
	 */
	public void setEvolucao(Long evolucao) {
		this.evolucao = evolucao;
	}

	public String getDataFormatadaDialogo() {
		return DateUtils.getDataFormatadaComHoraEMinutos(data);

	}

}
